pom.xml

```diff
@@ -5,7 +5,7 @@
     <name>Signature API-client Java</name>
     <groupId>no.digipost.signature</groupId>
     <artifactId>signature-api-client-java</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-STATUS-API-SNAPSHOT</version>
```

Comment:
Here is a maven command for setting version taking into account the current branch. May be useful (or not) for automating this:
```bash
mvn versions:set -DgenerateBackupPoms=false -DnewVersion=X.Y.$(git rev-parse --abbrev-ref HEAD)-SNAPSHOT```